frontend/src/components/pages/AdminDashboard.tsx

```diff
@@ -12,4 +12,4 @@ const AdminDashboard = (): React.ReactElement => {
   );
 };
 
-export default AdminDashboard;
+export default AdminDashboard;
```

Comment:
This change I think is getting rid of the last enter. Not a big deal, but it would be nice to remove it so this doesn't count as a file change.